winpty/__init__.py

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8 -*-
-
 """
 Pywinpty
 ========
 This package provides a Cython wrapper around winpty C++ library.
 """
+
 try:
```

Comment:
Now that we're getting the module __version__ without importing the module, we can delete this try-except block.